src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -1024,10 +1049,15 @@ export class IssueReporter extends Disposable {
 		const bugsUrl = this.getExtensionBugsUrl();
 		const extensionUrl = this.getExtensionRepositoryUrl();
 		// If given, try to match the extension's bug url
-		if (bugsUrl && bugsUrl.match(/^https?:\/\/github\.com\/(.*)/)) {
+		if (bugsUrl && bugsUrl.match(/^https?:\/\/github\.com\/([^\/]*)\/([^\/]*)\/([^\/]*)$/)) {
```

Comment:
This should also allow a repo like: `https://github.com/owner/repo` sometimes people do that.

Reply:
Also this regex matches discussions?
<img width="578" alt="image" src="https://github.com/microsoft/vscode/assets/2644648/46b4ff10-9fe9-46a3-8b59-ac5b79dacfd1">

Reply:
This is a real link:
http://github.com/microsoft/vscode-discussions/discussions

Reply:
so is this: https://github.com/community/community/discussions

Reply:
I feel like we should allow either:
* `https://github.com/org/repo`
* `https://github.com/org/repo/issues`

Reply:
hmm good point. maybe we just have the `bugsUrl` requirements for this case be:

`https://github.com/owner/repo` or `https://github.com/owner/repo/issues`